gamestate/user.go

```diff
@@ -45,20 +50,75 @@ func NewUserGateway(gfClient GFClient, loginSocket LoginSocket, gameSocket GameS
 	}
 }
 
+type GfAccountData struct {
+	Email    string
+	Password string
+	Locale   string
+	Name     string
+}
+
 // GameClientGateway provides methods to retrieve information about
 // the real game client.
 type GameClientGateway interface {
 	Version() string
 	Hash() string
 }
 
-func (ug *UserGateway) AuthenticateGFClient(user actions.User, serverLang string) (string, error) {
-	token, accountID, err := ug.gfClient.Authenticate(user, serverLang)
+func (ug *UserGateway) AuthenticateGFClient(jsonAccountPath string, jsonIdentityPath string) (string, error) {
+	content, err := ioutil.ReadFile(jsonAccountPath)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	identity_manager, err := gfclient_auth.NewIdentityManager(jsonIdentityPath)
+	if err != nil {
+		fmt.Println(err)
+	}
```

Comment:
You should return a proper error instead of printing it and continuing with the execution. Returning an error interrupting the execution of the function when something goes bad is the standard way of controlling the flow of the application in Go.

This would also be better suited to be in the main function of the bot, not in the noskit, as it pertains to the configuration and state of the bot more than game logic or state.